docs/index.rst

```diff
@@ -1,16 +1,49 @@
-.. dasbus documentation master file, created by
-   sphinx-quickstart on Wed Jan 22 21:51:51 2020.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 Welcome to dasbus's documentation!
 ==================================
 
+Dasbus is a DBus library written in Python 3, based on GLib and inspired by pydbus.
+The code used to be part of the `Anaconda Installer <https://github.com/rhinstaller/anaconda>`_
+project. It was based on the `pydbus <https://github.com/LEW21/pydbus>`_ library, but we replaced
+it with our own solution because of its inactivity. The dasbus library is a result of this effort.
```

Comment:
I think "but we decided to replace pydbus because its upstream development stalled" sounds better.

Reply:
Fixed in https://github.com/rhinstaller/dasbus/pull/29.